src/main/components/DataTree.jsx

```diff
@@ -4,7 +4,8 @@ import JsonTree from '../lib/treeView/jsonTree';
 import '../lib/treeView/jsonTree.css';
 
 const DataTree = (props) => {
-  const { treeCount, data } = props;
+  const { treeCount, data, options } = props;
+  const { onAdd, onEdit, onDelete, enableClipboard } = options;
```

Comment:
lets split this on separate lines to make the linter happy 
 const {
    onAdd, onEdit, onDelete, enableClipboard,
  } = options;